cmd/stackdriver-prometheus-sidecar/main.go

```diff
@@ -293,6 +300,20 @@ func main() {
 	}
 
 	cfg.projectIdResource = fmt.Sprintf("projects/%v", *projectId)
+	if cfg.apiOverride {
+		// manual.GenerateAndRegisterManualResolver generates a Resolver and a random scheme.
+		// It also registers the resolver. rb.InitialAddrs adds the addresses we are using
+		// to resolve GCP API calls to the resolver.
+		cfg.manualResolver, _ = manual.GenerateAndRegisterManualResolver()
+		cfg.manualResolver.InitialAddrs([]resolver.Address{
+			{Addr: "199.36.153.4:443"},
+			{Addr: "199.36.153.5:443"},
+			{Addr: "199.36.153.6:443"},
+			{Addr: "199.36.153.7:443"},
+		})
+	} else {
+		cfg.manualResolver = nil
```

Comment:
Nit, the else branch is not needed, pointers are initialized to `nil` by default.